GoMeet/src/main/webapp/js/createMeetingShared.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Takes all values from <input> tags on the page, and saves data to sessionStorage
+ * with 'name' of the <input> tag as key, and 'value' of the input tag as the value.
+ * Note:
+ *  - All <input> 'name's must be unique, otherwise the value will be overwritten.
+ *  - All <input> on the page are expected to be compulsory fields. Empty <input> will 
+ *  raise an alert requiring user to fill in every field.
+ * @returns true if data successfully stored, throws an Error otherwise
+ */
+function saveMeeting() {
+  let allInputs = document.getElementsByTagName('input');
+  let radioGroups = new Set();
+  let radioChecked = new Set(); 
+  for (let i = 0; i < allInputs.length; i++) {
+    let input = allInputs.item(i);
+    let key = input.name;
+    let value = input.value;
+    // If this is a radio button, add to the set of radio 
+    // groups if not there already.
+    if (input.type === 'radio') {
+      if (!radioGroups.has(input.name)) {
+        radioGroups.add(input.name);
+      }
+      // If the radio button is checked, add to checked set
+      if (input.checked) {
+        radioChecked.add(input.name);
+        sessionStorage.setItem(key, value);
+      }
+    }
+    else { // all other types of inputs
```

Comment:
nit: according to Google Javascript style guide "there is no line break after the brace if it is followed by else, catch, while, or a comma, semicolon, or right-parenthesis"

Reply:
Yep, fixed!